libqtile/window.py

```diff
@@ -485,9 +509,10 @@ def place(self, x, y, width, height, borderwidth, bordercolor,
             borderwidth=borderwidth,
         )
         if above:
-            kwarg['stackmode'] = StackMode.Above
+            self.to_relative_top(**kwarg)
+        else:
+            self.window.configure(**kwarg)
 
```

Comment:
I guess `to_relative_top` is doing more than it says on the tin, you are right. The purpose was to replace as many calls to `configure` as possible with it, since only the way through `to_relative_top` guarantees that the group notices changes to the window-stack.

The method is also called in `core/manager.py:755`. Passing in `kwargs` is not exactly pretty, but otherwise we would be calling `configure` twice - I don't know how it works in the backend, maybe there is a downside to that?

Reply:
I removed this. And I stopped worrying about calling `configure` to often, I'll leave that to #1924.